tests/load/gcp/prom-scrape-avalanche.tpl.conf

```diff
@@ -0,0 +1,30 @@
+#cloud-config
+
+write_files:
+- path: /run/start_avalanche
+  content: |
+    #!/bin/bash
+    set -euxo pipefail
+    for ((i = 1 ; i <= 20 ; i++)) ; do 
+        port=$((9000 + $i))
+        nohup avalanche \
+        --metric-count=${METRIC_COUNT} \
+        --label-count=10 \
+        --series-count=10 \
+        --value-interval=${VALUE_INTERVAL} \
+        --series-interval=36000000 \
+        --metric-interval=36000000 \
+        --port="$${port}" 1>/dev/null 2>&1 & 
+    done
```

Comment:
I wonder if you should use a similar approach as above with systemd. You can parameterise systemd so that something like `systemctl start avalance@1` and `systemctl start avalanche@2` would work :)

https://www.stevenrombauts.be/2019/01/run-multiple-instances-of-the-same-systemd-unit/